src/Components/Article_card/Article_card.jsx

```diff
@@ -1,5 +1,78 @@
-import React from "react";
+import React, { useEffect, usearticle, useState } from "react";
+import { useParams, Link } from "react-router-dom";
+import { getArticles } from "../../utils/API";
+import {
+  Wrapper,
+  Content,
+  Title,
+  Button,
+  AuthorTopic,
+  Votes,
+} from "./Article_card.styles";
 
-export default function Article_card() {
-  return <div>Article_card</div>;
-}
+const Article_card = () => {
+  const [article, setArticle] = useState([]);
+  const { article_id } = useParams();
+  const [err, setErr] = useState(null);
+
+  const deleteClick = () => {
+    alert("Do you really want to delete this?");
```

Comment:
I have to refresh the screen if I don't actually want to delete an article!

Reply:
Yeah, not implemented that yet, just a placeholder alert.